src/Libraries/UnitsUI/ViewModelCustomization.cs

```diff
@@ -0,0 +1,296 @@
+﻿using Dynamo.Controls;
+using Dynamo.Core;
+using Dynamo.Graph.Nodes;
+using Dynamo.UI.Commands;
+using Dynamo.ViewModels;
+using DynamoUnits;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitsUI
+{
+    public class ForgeConverterViewModel : NotificationObject
+    {
+        private readonly ForgeDynamoConvert dynamoConvertModel;
+        public DelegateCommand ToggleButtonClick { get; set; }
```

Comment:
can we move this away from the private fields and add summary

Reply:
Done.